opengrok-indexer/src/test/java/org/opengrok/indexer/analysis/haskell/HaskellXrefTest.java

```diff
@@ -122,8 +122,10 @@ public void sampleTest() throws IOException {
             expectedOutputSteam.close();
         }
 
-        String actual[] = new String(sampleOutputStream.toByteArray(), "UTF-8").split("\n");
-        String expected[] = new String(expectedOutputSteam.toByteArray(), "UTF-8").split("\n");
+        String actual[] = new String(sampleOutputStream.toByteArray(), "UTF-8").
```

Comment:
Why not use `split("\\r?\\n")` as suggested in <https://stackoverflow.com/questions/454908/split-java-string-by-new-line> ?